src/main/java/de/srendi/advancedperipherals/common/configuration/PeripheralsConfig.java

```diff
@@ -79,6 +88,12 @@ public class PeripheralsConfig implements IAPConfig {
     public final ForgeConfigSpec.IntValue poweredPeripheralMaxEnergyStorage;
     private final ForgeConfigSpec configSpec;
 
+    private static final List<String> chatBoxDefaultBannedCommands = Arrays.asList(
```

Comment:
I would also disable /give, /summon and /setblock to prevent item creation

Reply:
Maybe just check permission and make another option to disable all op commands

Reply:
that wiil be too expensive and too complex, well

Reply:
Another solution would be to execute the command via our own custom command like /execute. Something like `/apexecute <command>`
Then we could check for the permission when a user clicks on the message. Not sure if that's a  better option

Reply:
yeah good idea

Reply:
Do you want to do that in this pr or is that a thing for the future?

Reply:
I'll do it in this pr yes